exp/services/soroban-rpc/internal/methods/account.go

```diff
@@ -0,0 +1,52 @@
+package methods
+
+import (
+	"context"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/code"
+	"github.com/creachadair/jrpc2/handler"
+	"github.com/stellar/go/clients/horizonclient"
+)
+
+type AccountInfo struct {
+	ID       string ` json:"id"`
```

Comment:
nit: whitespace oddity? unsure if it matters

```suggestion
	ID       string `json:"id"`
```